### PR TITLE
fix: Top langs is not sorted by total sum size, fix #247

### LINKED ISSUE #247: Top langs is not sorted by total sum size
**Describe the bug**

Top langs is not sorted by total sum size.

As my account:

```js
debug repoNodes:  {
  Go: { name: 'Go', color: '#00ADD8', size: 378080 },
  Python: { name: 'Python', color: '#3572A5', size: 50925 },
  JavaScript: { name: 'JavaScript', color: '#f1e05a', size: 211391 },
  HTML: { name: 'HTML', color: '#e34c26', size: 37347 },
  CSS: { name: 'CSS', color: '#563d7c', size: 36145 },
  TypeScript: { name: 'TypeScript', color: '#2b7489', size: 38047 },
  Shell: { name: 'Shell', color: '#89e051', size: 13266 },
  Vue: { name: 'Vue', color: '#2c3e50', size: 10792 },
  'Objective-C': { name: 'Objective-C', color: '#438eff', size: 4416 },
  Smarty: { name: 'Smarty', color: null, size: 2370 },
  Java: { name: 'Java', color: '#b07219', size: 1252 },
  Makefile: { name: 'Makefile', color: '#427819', size: 8877 },
  HCL: { name: 'HCL', color: null, size: 566 },
  Dockerfile: { name: 'Dockerfile', color: '#384d54', size: 2151 },
  AMPL: { name: 'AMPL', color: '#E6EFBB', size: 180 }
}
```

**Expected behavior**

Top 5 langs should be:

```js
{
  Go: { name: 'Go', color: '#00ADD8', size: 378080 },
  JavaScript: { name: 'JavaScript', color: '#f1e05a', size: 211391 },
  Python: { name: 'Python', color: '#3572A5', size: 50925 },
  TypeScript: { name: 'TypeScript', color: '#2b7489', size: 38047 },
  HTML: { name: 'HTML', color: '#e34c26', size: 37347 }
}
```

**Screenshots / Live demo link (paste the github-readme-stats link as markdown image)**

current topLangs is:

```js
{
  Go: { name: 'Go', color: '#00ADD8', size: 378080 },
  Python: { name: 'Python', color: '#3572A5', size: 50925 },
  JavaScript: { name: 'JavaScript', color: '#f1e05a', size: 211391 },
  HTML: { name: 'HTML', color: '#e34c26', size: 37347 },
  CSS: { name: 'CSS', color: '#563d7c', size: 36145 }
}
```

**Additional context**

### PR DESCRIPTION
**Pull request by zcong1993**

### CODE REVIEW

File: src/fetchTopLanguages.js [hunk 1]
```diff
@@ -72,11 +71,12 @@ async function fetchTopLanguages(username) {
       };
     }, {});
 
-  const topLangs = Object.keys(repoNodes)
+  const topLangs = Object.values(repoNodes)
+    .sort((a, b) => b.size - a.size)
```
Comment (anuraghazra) [inline src/fetchTopLanguages.js:75]: We are already doing this on renderTopLanguages

https://github.com/anuraghazra/github-readme-stats/blob/a4486d032705f804965f22ebea4ed268e2af49a0/src/renderTopLanguages.js#L90

In #247 the sorting should happen while calculating the reduced language size not when we are slicing the languages.

Comment (zcong1993) [reply src/fetchTopLanguages.js:75]: The calculated repoNodes is an object, should I sort the object?

Comment (anuraghazra) [reply src/fetchTopLanguages.js:75]: I think what you are doing is fine, because we are already doing Object.values()

Comment (zcong1993) [reply src/fetchTopLanguages.js:75]: Yes, please merge it

**Comment by purplemysticx** [comment]
Why not merge this pull request?

**Pull request closed without merging**